### DISCUSSION TITLE
threading rendering

### DISCUSSION THREAD

**Discussion by Paraworker** [YaLTeR/niri#481] [Ideas]
Is there any plan for Niri to use dedicated thread for rendering? I noticed cosmic-comp has implemented this. It should be good for users with multiple high refresh rate monitors 🤔.

**Comment by YaLTeR** [comment]
I'll consider it if needed. It's a very complex change which should be approached with care. For now there's other, easier optimization potential left.

FWIW I'm also running multiple high resolution and refresh rate displays and semi regularly checking performance.
